src/openklant/conf/base.py

```diff
@@ -454,6 +490,19 @@
     "REMOTE_ADDR",
 )
 
+#
+# DJANGO-CORS-MIDDLEWARE
+#
+CORS_ALLOW_ALL_ORIGINS = config("CORS_ALLOW_ALL_ORIGINS", default=False)
+CORS_ALLOWED_ORIGINS = config("CORS_ALLOWED_ORIGINS", split=True, default=[])
+CORS_ALLOWED_ORIGIN_REGEXES = config(
+    "CORS_ALLOWED_ORIGIN_REGEXES", split=True, default=[]
+)
+# Authorization is included in default_cors_headers
+CORS_ALLOW_HEADERS = list(default_cors_headers) + config(
+    "CORS_EXTRA_ALLOW_HEADERS", split=True, default=[]
+)
+
```

Comment:
I notice that CORS_EXPOSE_HEADERS is in OpenZaak but not here nor in open-notificaties. Is that intentional?
```
CORS_EXPOSE_HEADERS = [
    "content-crs",
]
```

Reply:
This header is used by Open Zaak (and I think Objects) to determine the coordinate system in a geospatial context (https://docs.ogc.org/is/18-058/18-058.html#_coordinate_reference_system_information_independent_of_the_feature_encoding). Open Klant and Open Notificaties do not use this header, so the `CORS_EXPOSE_HEADERS` is not needed for that reason.